.build/tasks/generateHelp.PlatyPS.build.ps1

```diff
@@ -41,26 +48,162 @@ Task UpdateHelp {
+    )
+
+    Write-Build -Color DarkGray -Text "$generateHelpCommands"
+    $sb = [ScriptBlock]::create(($generateHelpCommands -join "`r`n"))
+    Write-Build -Color DarkGray -Text "$sb"
+
+    $pwshPath = (Get-Process -Id $PID).Path
+    &$pwshPath -c $sb -ep 'ByPass'
 }
 
+Task UpdateHelp {
+    if ([System.String]::IsNullOrEmpty($ProjectName))
+    {
+        $ProjectName = Get-SamplerProjectName -BuildRoot $BuildRoot
+    }
+
+    if ([System.String]::IsNullOrEmpty($SourcePath))
+    {
+        $SourcePath = Get-SamplerSourcePath -BuildRoot $BuildRoot
+    }
+
+    $OutputDirectory = Get-SamplerAbsolutePath -Path $OutputDirectory -RelativeTo $BuildRoot
+    $PesterOutputFolder = Get-SamplerAbsolutePath -Path $PesterOutputFolder -RelativeTo $OutputDirectory
 
-Task GenerateMamlFromMd {
+    "`tProject Name             = '$ProjectName'"
+    "`tOutput Directory         = '$OutputDirectory'"
+    "`tPester Output Folder     = '$PesterOutputFolder"
+
+    if ($VersionedOutputDirectory)
+    {
+        # VersionedOutputDirectory is not [bool]'' nor $false nor [bool]$null
+        # Assume true, wherever it was set
+        $VersionedOutputDirectory = $true
+    }
+    else
+    {
+        # VersionedOutputDirectory may be [bool]'' but we can't tell where it's
+        # coming from, so assume the build info (Build.yaml) is right
+        $VersionedOutputDirectory = $BuildInfo['VersionedOutputDirectory']
+    }
+
+    $GetBuiltModuleManifestParams = @{
+        OutputDirectory          = $OutputDirectory
+        BuiltModuleSubDirectory  = $BuiltModuleSubDirectory
+        ModuleName               = $ProjectName
+        VersionedOutputDirectory = $VersionedOutputDirectory
+        ErrorAction              = 'Stop'
+    }
+
+    $builtModuleBase = Get-SamplerBuiltModuleBase @GetBuiltModuleManifestParams
+    "`tBuilt Module Base        = '$builtModuleBase'"
+
+    $builtModuleManifest = Get-SamplerBuiltModuleManifest @GetBuiltModuleManifestParams
+    "`tBuilt Module Manifest    = '$builtModuleManifest'"
+
+    if ($builtModuleRootScriptPath = Get-SamplerModuleRootPath -ModuleManifestPath $builtModuleManifest)
+    {
+        $builtModuleRootScriptPath = (Get-Item -Path $builtModuleRootScriptPath -ErrorAction SilentlyContinue).FullName
+    }
+
+    "`tBuilt ModuleRoot script  = '$builtModuleRootScriptPath'"
+
+    $ModuleVersion = Get-BuiltModuleVersion @GetBuiltModuleManifestParams
+    $ModuleVersionObject = Split-ModuleVersion -ModuleVersion $ModuleVersion
+    $ModuleVersionFolder = $ModuleVersionObject.Version
+    $preReleaseTag       = $ModuleVersionObject.PreReleaseString
+
+    "`tModule Version           = '$ModuleVersion'"
+    "`tModule Version Folder    = '$ModuleVersionFolder'"
+    "`tPre-release Tag          = '$preReleaseTag'"
+
+    $HelpFolder = Get-SamplerAbsolutePath -Path $HelpFolder -RelativeTo $SourcePath
+    "`tHelp Folder Path         = '$HelpFolder'"
```

Comment:
Depending on what is determined in the above comment, this would also need to be modified.

Reply:
should be fixed.